buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -269,34 +270,36 @@ class PerformanceTestPlugin : Plugin<Project> {
             createDistributedPerformanceTestTask(name, clazz, performanceSourceSet, prepareSamplesTask).configure(configure)
         }
 
+        val channelSuffix = if (OperatingSystem.current().isLinux) "" else "-${OperatingSystem.current().name.toLowerCase()}"
+
         create("distributedPerformanceTest", DistributedPerformanceTest::class) {
             includeCategories(performanceRegressionTestCategory)
             excludeCategories(slowPerformanceRegressionTestCategory)
-            channel = "commits"
+            channel = "commits$channelSuffix"
```

Comment:
Did you update the performance database contents? If you don't, then we'll lose all the history, right?